Editor/Window/AnywherePage.cs

```diff
@@ -39,13 +42,21 @@ public AnywherePage(VisualElement container, StateManager stateManager)
             var computeInputContainer = uxml.Q("AnywherePageComputeTitle");
             var computeInput =
                 new RegisterComputeInput(computeInputContainer, stateManager);
-            _launchButton = uxml.Q<Button>("AnywherePageLaunchServerButton");
-            _launchButton.RegisterCallback<ClickEvent>(_ =>
+            _launchServerButton = uxml.Q<Button>("AnywherePageLaunchServerButton");
+            _launchServerButton.RegisterCallback<ClickEvent>(_ =>
             {
                 EditorUserBuildSettings.SwitchActiveBuildTarget(NamedBuildTarget.Server,
                     EditorUserBuildSettings.selectedStandaloneTarget);
                 EditorApplication.EnterPlaymode();
             });
+
+            _configureClientButton = uxml.Q<Button>("AnywherePageConfigureClientButton");
+            _configureClientButton.RegisterCallback<ClickEvent>(_ =>
+            {
+                GameLiftClientSettings gameLiftClientSettings = AssetDatabase.LoadAssetAtPath<GameLiftClientSettings>("Assets/Settings/GameLiftClientSettings.asset");
```

Comment:
`AssetsPath.cs` which contains this static string is in the SampleGame assembly, which isn't accessible from here. Hardcoding for now.

Reply:
We decided to push this to a separate PR to unblock this merge